README.md

```diff
@@ -119,4 +119,21 @@ docker run -d -p 'Port you want to use':5000 --env-file .\.env --mount type=bind
 
 Once complete you can go to http://localhost:5000/ to view the page or http://localhost:'Port you want to use'/  replacing 'Port you want to use' with your defined port number
 
+# to Run with selenium and firefox for end to end tests add the following
+```bash
+pip install selenium
```

Comment:
Rather than using pip, we should track it with poetry like our other dependencies!
`poetry add selenium --dev`

One advantage of this is that this will add it to your `pyproject.toml` and `poetry.lock` files, which will mean that future users pick up the dependencies automatically when they run `poetry install` rather than having to manually install each & every package

Reply:
updated